erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py

```diff
@@ -3181,6 +3175,12 @@ def test_gain_loss_with_advance_entry(self):
 		)
 
 
+def enable_discount_accounting(enable=1):
```

Comment:
There's already a simpler/reliable way to do this.
https://github.com/frappe/erpnext/blob/c42dcaa6cff5c87a10d58854cd10089f5ba53efb/erpnext/stock/doctype/stock_entry/test_stock_entry.py#L769